catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/security/AccessControlPolicyExtensionTest.java

```diff
@@ -193,23 +193,10 @@ public void testSystemShouldImplyAll() {
     assertThat(after, is(ImmutableList.of()));
   }
 
-  @Test
-  public void testOwnerShouldImplAll() {
```

Comment:
Do we not want this test anymore?

Reply:
We do, I just renamed it `https://github.com/codice/ddf/blob/6bddfc3f83e21264c089bbf58ecbd376b9a598e7/catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/security/AccessControlPolicyExtensionTest.java#L106`